app/views/find_local_council/one_council.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render layout: 'base_page', locals: {
+  title: 'Find Your Local Council'
```

Comment:
We don’t generally capitalise page titles, probably want sentence case (e.g. `Find your local council`).